src/main/resources/application.yaml

```diff
@@ -45,3 +45,10 @@ scheduling:
     pending-notifications:
       delay: ${PENDING_NOTIFICATIONS_TASK_DELAY_IN_MS}
       enabled: ${PENDING_NOTIFICATIONS_TASK_ENABLED}
+
+# clients region
+
+s2s-auth:
+  url: ${S2S_URL:http://localhost:4552}
```

Comment:
do we want this default (`http://localhost:4552`) to be here? @nitinprabhu

Reply:
ah yes good point once we have it chart then we can remove it. If chart config is not done yet then we will to remove it when it's there

Reply:
@apereverzin Please remove it once u do the chart config. Ideally we do the chart config first

Reply:
i also added as a note during, i believe, first review :wink:

Reply:
@nitinprabhu Chart config is done so removed default value